src/events/deposits.upsert.service.ts

```diff
@@ -131,4 +132,23 @@ export class DepositsUpsertService {
 
     return deposits;
   }
+
+  async mismatchedDepositCount(): Promise<number> {
+    const result = await this.prisma.$queryRawUnsafe<{ count: number }[]>(
+      `
+      SELECT
+        COUNT(*) AS count
+      FROM
+        blocks
+      RIGHT OUTER JOIN
+        deposits
+      ON blocks.hash = deposits.block_hash
+      WHERE blocks.hash IS NULL OR blocks.main <> deposits.main
+      `,
+    );
```

Comment:
We should figure out how to run this against the read only slave.

Reply:
```
Finalize Aggregate  (cost=312701.30..312701.30 rows=1 width=8)
   ->  Gather  (cost=312701.20..312701.30 rows=1 width=8)
         Workers Planned: 1
         ->  Partial Aggregate  (cost=311701.20..311701.20 rows=1 width=8)
               ->  Parallel Hash Left Join  (cost=3870.07..311022.97 rows=1356454 width=0)
                     Hash Cond: ((deposits.block_hash)::text = (blocks.hash)::text)
                     Filter: ((blocks.hash IS NULL) OR (blocks.main <> deposits.main))
                     ->  Parallel Seq Scan on deposits  (cost=0.00..290953.20 rows=5767399 width=66)
                     ->  Parallel Hash  (cost=3661.11..3661.11 rows=59704 width=66)
                           ->  Parallel Seq Scan on blocks  (cost=0.00..3661.11 rows=59704 width=66)
```

Reply:
It looks like `queryRawUnsafe` is already set to query from the read instance on production https://github.com/iron-fish/ironfish-api/blob/master/src/prisma/prisma.service.ts#L33